model/schema/affectedGenomicModel.yaml

```diff
@@ -182,7 +185,12 @@ slots:
   parental_populations:
```

Comment:
Do we still need this slot with this change?

Reply:
@sierra-moxon 
The parental_population slot is for the record in the database, the parental_population_curies slot is only for the DTO class for submitted AGMs. They are both needed, I think.